addons/adminMessages/XEH_preInit.sqf

```diff
@@ -7,3 +7,6 @@ PREP_RECOMPILE_START;
 PREP_RECOMPILE_END;
 
 ADDON = true;
+
+
+[QGVAR(EH_recieveMessage), FUNC(recieveMessage)] call CBA_fnc_addEventHandler;
```

Comment:
spelling error in the gvars, it's `receive` instead of `recieve`

Reply:
🙈